Leanplum-SDK/Classes/Leanplum.h

```diff
@@ -241,6 +242,11 @@ NS_SWIFT_NAME(setApiHostName(_:servletName:ssl:));
  */
 + (void)setVerboseLoggingInDevelopmentMode:(BOOL)enabled;
 
+/**
```

Comment:
We might want to add an option to not print anything based on the developer's choice.